frontend/composables/useCustomFetch.ts

```diff
@@ -38,6 +38,7 @@ export function useCustomFetch () {
     const { public: { apiClient, legacyApiClient, apiKey, domain, logIp }, private: pConfig } = runtimeConfig
     const path = map.mock ? `${pathName}.json` : map.getPath?.(pathValues) || map.path
     let baseURL = map.mock ? '../mock' : map.legacy ? legacyApiClient : apiClient
+    const ssrSecret = process.env.NUXT_PRIVATE_SSR_SECRET
```

Comment:
i believe this approach will only allow us to specify the secret as runtime environment variables, but we need to be able to specifiy it both during runtime (for our own deployments) and during compile time (for deployments via cloudflare pages)

Reply:
Understood.